installer/install.iss

```diff
@@ -1076,11 +1080,20 @@ end;
 
 function DetectNetFxVersion:Cardinal;
 begin
+#if INSTALLER_FILENAME_SUFFIX=='arm64'
+    // Windows ARM64 is only available starting with Windows 10 1803,
+    // which has .NET Framework 4.7.2 built-in, but only 32-bit (which
+    // causes the registry check below to return 0). Therefore, we
+    // hardcode the version here.
+    // https://docs.microsoft.com/en-us/dotnet/framework/migration-guide/versions-and-dependencies
+    Result:=461808;
```

Comment:
Could AMD64 Windows also have only a 32 bit .NET framework installed? Should we try to additionally discover 32 bit .NET framework on 64 bit systems?

This workaround works for now, but will break once GCM core [moves their windows build to .NET 5](https://github.com/microsoft/Git-Credential-Manager-Core/issues/113).

Reply:
Good point.

Is there maybe a way to check the 32-bit version of the registry as a fall-back, once the 64-bit test failed?

Reply:
It [looks](https://jrsoftware.org/ishelp/index.php?topic=registrysection) as if `HKLM32` and `HKLM64` can be used for this purpose, and obviously their use should be guarded by `if IsWin64() then [...]`.

Reply:
> Could AMD64 Windows also have only a 32 bit .NET framework installed?

I think there's a very small chance this will happen, because Windows 10 has both 32-bit and 64-bit .NET framework built-in on AMD64. When [downloading the runtime manually](https://dotnet.microsoft.com/download/dotnet-framework/net472), it installs 32-bit and 64-bit in one go.

However, it's rather easy to build the check with the suggestions @dscho mentioned. I got it to work, just updated this PR accordingly.

I just downloaded a Windows 7 32-bit VM for testing ([from the official Microsoft website](https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/)), and with .NET framework installed it works:

![image](https://user-images.githubusercontent.com/17739158/115296562-65fb5c80-a15b-11eb-8812-257f781a3778.png)

When I remove .NET framework it correctly greys out the options:

![image](https://user-images.githubusercontent.com/17739158/115296766-a78c0780-a15b-11eb-8c84-93f249a2e53e.png)

Let me know if there's anything you want me to change in the logic 👍🏼

Reply:
I don't like how it has none of the radio buttons checked, but next still enabled in the no .NET case, but that's entirely unrelated to the changes and would happen like that on otherintel based cpus as well. It looks good to me.